src/components/modal/v2/parts/views/ShortTerm/Content.jsx

```diff
@@ -12,11 +13,19 @@ import { useServerData } from '../../../lib/providers';
 import { currencyFormat } from '../../../lib/hooks/currency'; // Remove .00 cents from formated min and max
 
 export const ShortTerm = ({
-    content: { instructions, linkToProductList, estimatedInstallments, disclosure, donutTimestamps, learnMoreLink },
-    productMeta: { qualifying, periodicPayment },
+    content: {
+        instructions,
+        linkToProductList,
+        estimatedInstallments,
+        preapproval: { preapprovalDisclaimerHeadline, preapprovalDisclaimerBody },
+        disclosure,
+        donutTimestamps,
+        learnMoreLink
+    },
+    productMeta: { qualifying, periodicPayment, preapproved },
     openProductList
 }) => {
-    const { views } = useServerData();
+    const { views, country } = useServerData();
```

Comment:
```suggestion
    const { views, country } = useServerData();
    const preapprovalDisclaimerHeadline = preapproval?.preapprovalDisclaimerHeadline;
    const preapprovalDisclaimerBody = preapproval?.preapprovalDisclaimerBody;
```
Similar reasoning as in `BodyContent.jsx`, we can check to make sure we can safely grab this content if available.